source/features/feature-descriptions.tsx

```diff
@@ -0,0 +1,152 @@
+import parseBackticks from '../github-helpers/parse-backticks';
+
+interface Commit {
+	messageHeadline: string;
+	messageHeadlineHTML: string;
+	commitUrl: string;
+	committedDate: string;
+}
+
+const getFeatureHistory = cache.function(async (fileName: string): Promise<Commit[]> => {
+	const {repository} = await api.v4(`
+		repository() {
+			defaultBranchRef {
+				target {
+					...on Commit {
+						history(first:100, path: "source/features/${fileName}") {
+							nodes {
+								messageHeadline
+								messageHeadlineHTML
+								commitUrl
+								committedDate
+							}
+						}
+					}
+				}
+			}
+		}
+	`);
+	return repository.defaultBranchRef.target.history.nodes;
+}, {
+	maxAge: {days: 1},
+	staleWhileRevalidate: {days: 4},
+	cacheKey: ([fileName]): string => __filebasename + ':' + fileName
+});
+
+function getCommitUrl(commit: Commit): string {
+	const [, pullRequestUrl] = /<a[^>]+href="([^"]+)">/.exec(commit.messageHeadlineHTML) ?? [];
+	return pullRequestUrl ?? commit.commitUrl;
+}
+
+async function getHistoryDropdown(featureName: string): Promise<Element> {
+	let history = await getFeatureHistory(featureName + '.tsx');
+
+	if (history.length === 0) {
+		// Feature might be CSS-only
+		history = await getFeatureHistory(featureName + '.css');
+	}
+
+	const filteredHistory = history.filter((commit: Commit) => !/^Meta|^Document|^Readme|^Lint|^Update.+dependencies/.test(commit.messageHeadline));
+	const skippedCommitsCount = history.length - filteredHistory.length;
+
+	const historyUrl = new GitHubURL(location.href);
+	historyUrl.assign({route: 'commits'});
+
+	return (
+		<details className="dropdown details-reset details-overlay d-inline-block">
+			<summary className="text-gray d-inline" aria-haspopup="true">
+				Feature history
+				<div className="dropdown-caret ml-1"/>
+			</summary>
+
+			<div className="dropdown-menu dropdown-menu-s" style={{width: 400}}>
+				<ul className="overflow-y-auto" style={{maxHeight: '60vh'}}>
+					{filteredHistory.map(commit => (
+						<li>
+							<a className="dropdown-item" href={getCommitUrl(commit)} title={commit.messageHeadline}>
+								{parseBackticks(commit.messageHeadline)}
+								<div className="text-small">
+									<relative-time datetime={commit.committedDate}/>
+								</div>
+							</a>
+						</li>
+					))}
+					{skippedCommitsCount > 0 && (
+						<>
+							<li className="dropdown-divider" role="separator"/>
+							<li>
+								<a className="dropdown-item" href={String(historyUrl)}>
+									View full history (+{skippedCommitsCount} commits)
+								</a>
+							</li>
+						</>
+					)}
+				</ul>
+			</div>
+		</details>
+	);
+}
+
+function getConversationsLink(featureName: string): Element {
+	const searchUrl = new URL('https://github.com/sindresorhus/refined-github/issues');
+	searchUrl.searchParams.set('q', `"${featureName}" sort:updated-desc`);
+
+	return <a className="ml-3" href={String(searchUrl)}>Conversations</a>;
```

Comment:
Feels like an overkill just to avoid manually escaping 2 characters. I'm sure Sindre disagrees, but 😜
```suggestion
	const url = '/sindresorhus/refined-github/issues?q=${featureName}+sort%3Aupdated-desc';
	return <a className="ml-3" href={url}>Conversations</a>;
```